trajopt/src/collision_terms.cpp

```diff
@@ -528,8 +876,43 @@ void CastCollisionEvaluator::CalcCollisions(const DblVec& x, tesseract_collision
             }
           }
 
+          // Dont include contacts at the fixed state
+          auto end =
+              std::remove_if(pair.second.begin(), pair.second.end(), [=](const tesseract_collision::ContactResult& r) {
+                switch (type_)
+                {
+                  case CollisionExpressionEvaluatorType::START_FREE_END_FREE:
+                  {
+                    return false;
+                  }
+                  case CollisionExpressionEvaluatorType::START_FIXED_END_FREE:
+                  {
+                    if (r.cc_type[0] == tesseract_collision::ContinuousCollisionType::CCType_Time0)
+                      return true;
+
+                    if (r.cc_type[1] == tesseract_collision::ContinuousCollisionType::CCType_Time0)
+                      return true;
```

Comment:
Should CCTYPE_NONE be removed?

Reply:
Also, do these enums follow the correct style?

Reply:
Are you suggesting removing it from the enum class?

Reply:
No, should it return true if it is that type.

Reply:
Right now CCType_None is to represent objects that are static in the world. You should never have a case where you checking two shapes that are of CCType_None which is why I only check for the CCType_Time0 or CCType_Time1 depending on the condition.